package.json

```diff
@@ -118,26 +118,26 @@
     "dtrace-provider": "^0.8.1"
   },
   "devDependencies": {
-    "autocannon": "0.16.5",
-    "autocannon-compare": "0.3.0",
+    "autocannon": "^0.16.5",
+    "autocannon-compare": "^0.3.0",
     "chai": "^4.1.2",
     "cover": "^0.2.9",
     "coveralls": "^2.13.3",
     "documentation": "^5.3.3",
-    "eslint": "^4.10.0",
-    "eslint-config-prettier": "^2.8.0",
+    "eslint": "^4.13.1",
+    "eslint-config-prettier": "2.9.0",
```

Comment:
should this be floating?

Reply:
I did it on purpose to linter issues from prettier updates.
Whish I could comment in `package.json` :)